ivy/functional/frontends/jax/numpy/logic.py

```diff
@@ -218,3 +218,14 @@ def iscomplex(x: any):
 @to_ivy_arrays_and_back
 def iscomplexobj(x):
     return ivy.is_complex_dtype(ivy.dtype(x))
+
+
+@to_ivy_arrays_and_back
+def fromfunction(function, shape, *, dtype=float, **kwargs):
+    arr = ivy.empty(shape, dtype=dtype)
+    for indices in ivy.ndindex(shape):
+        f_indices = indices
+        if len(shape) > 1 and (indices[-1] < 1):
+            f_indices = indices[:-1]
+        ivy.set_nest_at_index(arr, f_indices, function(*indices))
```

Comment:
You are supposed to pass in the **kwargs into the `function` like so: `function(*indices, **kwargs)`

Reply:
😓